packages/design-system-tailwind-preset/src/colors.ts

```diff
@@ -6,6 +6,9 @@ export const colors = {
     alternative: 'var(--color-background-alternative)',
     'alternative-hover': 'var(--color-background-alternative-hover)',
     'alternative-pressed': 'var(--color-background-alternative-pressed)',
+    muted: 'var(--color-background-muted)',
+    'muted-hover': 'var(--color-background-muted-hover)',
+    'muted-pressed': 'var(--color-background-muted-pressed)',
```

Comment:
Adding the new `background-muted` to the Tailwind preset. A test initially failed when the new `background-muted` CSS variables were added to the design tokens, which is great!  

![Screenshot 2024-11-29 at 9 03 29 AM](https://github.com/user-attachments/assets/d4e886b3-5c9d-4c47-b6f1-67eceec997ae)